src/Symfony/Component/Notifier/Bridge/Iqsms/README.md

```diff
@@ -0,0 +1,12 @@
+Iqsms Notifier
+==============
+
+Provides Iqsms integration for Symfony Notifier.
```

Comment:
Can you add a note about the DSN format (like done in some other providers)?

Reply:
Thank for your review. Note already added.